bulk-targeted-sponsorship/README.md

```diff
@@ -0,0 +1,381 @@
+  … more
+]
+  relationships: Array(819) [Object, Object, Object, Object, Object, Object, Object, Object, Object, Object, Object, Object, Object, Object, Object, Object, Object, Object, Object, Object, …]
+}
+```
+
+You can get to this same SBOM JSON by calling 
+the [GitHub API SBOM endpoint](https://docs.github.com/en/rest/dependency-graph/sboms?apiVersion=2022-11-28).
+
+The GitHub API is a way to create an inventory of dependencies across all of your repositories. As explained in the pseudo code below, you can call the 
+GitHub API to get a list of all repositories in all of your organizations, and
+then one by one call the SBOM API endpoint for each repository. You can then
+create a data structure for how many times you see each dependency.
+
+_**Pseudo code of how to get inventory of your enterprise's dependencies**_
+```Pseudo code
+
+dependencyInventory = {
+   dependencyNameA: count,
+   dependencyNameB: count
+   ... 
+}
+
+for eachOrganization in listofOrganizations:
+    for eachRepo in eachOrganization: 
+        SBOM = call GitHubAPI(eachRepo)
+        listOfDependencies = parse(SBOM)
+        for eachDependency in listOfDependencies:
+            addToDependencyInventory(eachDependency)
+
+return dependencyInventory
+
+```
+
+## Collecting metadata that describes your dependencies
+### _Metadata helps you know information about dependencies without reading_
+
+To filter that inventory of dependencies down to those you want to sponsor, you
+will need metadata information about each package and its source repository.
+The alternative, already knowing about each dependency or reading through
+source repositories doesn't scale to the dependency usage of even a small 
+organization, let alone Microsoft. Although rates vary by language,
+an organization with several hundred repositories will likely have many 
+thousands of dependencies.
+
+### Advanced: collecting metadata the describes your dependencies
+
+For selecting which dependencies to sponsor in this round,
+Microsoft looked at a number of dimensions leveraging different
+data sources stored in different [Kusto tables](https://learn.microsoft.com/en-us/azure/data-explorer/kusto/query/). 
+The main source of this data was the 
+[packages endpoint of Ecosyste.ms](https://packages.ecosyste.ms/), which provides
+metadata that describes package and source repository public usage rates, 
+dependency and dependent relationships, activity, etc.
+We also used data from [ClearlyDefined.io](https://clearlydefined.io/)
+for license information, experimented with the 
+[OSSF risk API](https://riskapi.ashydesert-4ee1f08e.westus3.azurecontainerapps.io/apidocs/) protype for information on dependency dependency's average age, and integrated
+internal metadata on how dependencies were used.
+
+### Basic:  collecting metadata the describes your dependencies
```

Comment:
I assume this should be `that` and not `the`

Reply:
fixed